src/functions/genfrac.js

```diff
@@ -321,3 +335,137 @@ defineFunction({
         }, parser.mode);
     },
 });
+
+const stylArray = ["display", "text", "script", "scriptscript"];
+
+const delimFromValue = function(delimString: string): string | null {
+    let delim = null;
+    if (delimString.length > 0) {
+        delim = delimString;
+        delim = delim === "." ? null : delim;
+    }
+    return delim;
+};
+
+defineFunction({
+    type: "genfrac",
+    names: ["\\genfrac"],
+    props: {
+        numArgs: 6,
+        greediness: 6,
+        argTypes: ["math", "math", "size", "text", "math", "math"],
```

Comment:
Apparently TeX accepts any math for the first two args, but if it's anything other than a delimiter the results end up being weird.